harshita/level1/medicalissue.cpp

```diff
@@ -0,0 +1,31 @@
+
+#include<iostream>
+using namespace std;
+
+
+int main()
+{
+	int numclass, attend, total;
+	char issue,Y,N;
+	cout << "Number of Classes: ";
+	cin >> numclass;
+	cout << "Number of Classes attended: ";	
+	cin >> attend;
+	total= (attend / numclass)* 100;
+	cout << "Medical Issue(PICK Y/N): ";
+	cin >> issue;
+	if(total< 75){
```

Comment:
It's wrong let say total is 60  & issue is Y. Then it should allow but here in your code as soon as it read first if condition it will say not allowed. So better code will be 
```
if(total >= 75) 
cout << "Yes";
else if(issue == 'Y')
cout << "Yes";
else
cout << "NO";
```